.meta/sinks/gcp_stackdriver_logging.toml

```diff
@@ -0,0 +1,97 @@
+[sinks.gcp_stackdriver_logging]
+title = "GCP Stackdriver Logging"
+batch_max_size = 5242880
+batch_timeout_secs = 1
+beta = true
+buffer = true
+common = true
+delivery_guarantee = "best_effort"
+function_category = "transmit"
+healthcheck = true
+egress_method = "batching"
+input_types = ["log"]
+request_rate_limit_duration_secs = 1
+request_rate_limit_num = 100
```

Comment:
Is this aligned with API limits or just an arbitrary limit?

Reply:
This rate limit is aligned with the API, yes, although the docs have a typo (too few zeros), which I have now fixed.